dashboard/db/migrate/20160510192122_populate_hashed_email_in_users.rb

```diff
@@ -0,0 +1,18 @@
+# A migration that populates (if not already populated) a user's hashed_email
+# for users having an (non-empty) email address.
+
+class PopulateHashedEmailInUsers < ActiveRecord::Migration
+  def change
+    reversible do |direction|
+      direction.up do
+        User.with_deleted.
+            where('email IS NOT NULL AND email <> ""').
```

Comment:
do the chained where clauses result in an efficient query (versus a single where clause with everything anded together?)